src/Phase.jl

```diff
@@ -28,7 +28,7 @@ include("Reaction.jl")
     veckineticsinds::Array{Int64,1}
     vecthermo::NASAvec
     otherreactions::Array{ElementaryReaction,1}
-    electronchange::Union{Nothing,W3}
+    electronchange::W3
```

Comment:
Can you squash this to the previous commit that added `Union{Nothing,W3}`

Reply:
done